lib/active_merchant/billing/gateways/realex.rb

```diff
@@ -70,9 +71,9 @@ def refund(money, authorization, options = {})
         commit(request)
       end
 
-      def credit(money, authorization, options = {})
-        ActiveMerchant.deprecated CREDIT_DEPRECATION_MESSAGE
```

Comment:
Have you looked yet into whether we need to do anything special now that we're removing support for the way this deprecated request originally operated? (i.e., it's changing from a refund for a particular authorization to a general credit).

Reply:
Yes, I have asked Connect about this issue. https://spreedly.slack.com/archives/CF080246R/p1563548513083000

> IMO it's safe enough, nothing special needed except a mention in the commit, maybe make the commit title descriptive that it's a re-implementation (Re-implement credit as general credit, or something)

Reply:
👍